src/rating/Rating.js

```diff
@@ -138,25 +153,57 @@ export default class Rating extends Component {
 
   getCurrentRating() {
     const { value } = this.state;
-    const { imageSize, ratingCount } = this.props;
+    const { fractions, imageSize, ratingCount } = this.props;
     const startingValue = ratingCount / 2;
     let currentRating = 0;
 
     if (value > ratingCount * imageSize / 2) {
       currentRating = ratingCount;
     } else if (value > imageSize) {
-      currentRating = Math.ceil(startingValue + value / imageSize);
+      currentRating = startingValue + value / imageSize;
+      currentRating = !fractions
+        ? Math.ceil(currentRating)
+        : +currentRating.toFixed(fractions);
```

Comment:
Simply put, `toFixed` returns a string representation of the number rounded to the specified decimal places. With `fractions={13}` and a rating of `2.5`, you get `2.5000000000000`.

I wanted a way to display Facebook ratings, which contain 1 decimal place. But I saw no reason to limit the number of decimal places supported by `toFixed`.